### PR TITLE
id nil causes app to crash

### PR DESCRIPTION
**Pull request by aimroboticsbying**
bug introduced from my last pull request.

### CODE REVIEW

File: ios/RCTMGL/RCTMGLMapTouchEvent.m [hunk 1]
```diff
@@ -49,6 +49,7 @@ + (RCTMGLMapTouchEvent*)_fromPoint:(CGPoint)point withMapView:(MGLMapView *)mapV
 {
     RCTMGLMapTouchEvent *event = [[RCTMGLMapTouchEvent alloc] init];
     event.type = eventType;
+    event.id = @"";
```
Comment (mfazekas) [inline ios/RCTMGL/RCTMGLMapTouchEvent.m:52]:
```suggestion
    event.id = [NSNull null];
```

Comment (aimroboticsbying) [reply ios/RCTMGL/RCTMGLMapTouchEvent.m:52]: id type is NSString. Also NSNull probably is bad for geoJSONDictionary at line 25.

Comment (mfazekas) [reply ios/RCTMGL/RCTMGLMapTouchEvent.m:52]: yes your right, NSNull is probably not a good idea.

What about just not adding the id attribute to the attributes if it's nil?! 
I don't like the idea of adding a "id" with an empty  string, when there is no id.

https://github.com/react-native-mapbox-gl/maps/blob/fd4e030210ad126bdf0e36c66526a1b855a063ea/ios/RCTMGL/RCTMGLMapTouchEvent.m#L21

Comment (aimroboticsbying) [reply ios/RCTMGL/RCTMGLMapTouchEvent.m]: Sure

**Pull request merged**